packages/react/src/components/CommandCenter/Input.tsx

```diff
@@ -0,0 +1,52 @@
+import { useCallback, useEffect, useState, forwardRef } from 'react';
+import styles from './styles.module.css';
+
+type InputProps = {
+    value: string;
+    onChange: (value: string) => void;
+    placeholder: string;
+    label: string;
+};
+
+export const Input = forwardRef<HTMLInputElement, InputProps>(function Input({
```

Comment:
forwardRef to add keyboard event handlers to the input ref. 

We wan't to keep concerns separate and the keyboard event handlers in this case aren't really related to input component's logic thus it's better to use a forward ref here. 

If it were just `Enter`, we could have added the keybinding within the component and skipped the `forwardRef` part